internal/host/hostcommands/instruction_manager_test.go

```diff
@@ -205,6 +208,63 @@ var _ = Describe("instruction_manager", func() {
 		})
 	})
 
+	Context("Disable Steps verification", func() {
+		BeforeEach(func() {
+			os.Setenv("DISABLED_STEPS", "")
```

Comment:
Modifying os environments could disrupt parallel tests.
You should just modify the config without the OS environment dependency.

Reply:
Got it - Will do.